examples/debhelper/debathena-transform-example-1.0/debian/compat

```diff
@@ -1 +1 @@
-7
+9
```

Comment:
I suppose that by bumping the compat level to 9 we potentially cause problems for people trying to use our latest stuff on older releases, but it's also not hard to work around.

Reply:
9 is supported as far back as wheezy and trusty, so we're fine (continued users of squeeze or precise or something are on their own). I did mention this in the commit message but that's not super obvious from the PR interface.